tyrian/js/src/main/scala/tyrian/Cmd.scala

```diff
@@ -95,9 +104,12 @@ object Cmd:
     def empty: Cmd[F, Msg]                                   = Cmd.None
     def combine(a: Cmd[F, Msg], b: Cmd[F, Msg]): Cmd[F, Msg] = Cmd.merge(a, b)
 
+  // XXX - How to do equality?
+  /*
   given [F[_]: Applicative, Msg: Eq](using ev: Eq[F[Option[Msg]]]): Eq[Cmd[F, Msg]] with
     def eqv(x: Cmd[F, Msg], y: Cmd[F, Msg]): Boolean =
       CmdHelper.cmdToTaskList(x) === CmdHelper.cmdToTaskList(y)
+  */
```

Comment:
I take it this doesn't work now, what does the error look like?

Reply:
Two commands are equal when they produce equal actions. So, to compare two commands, translate them to their actions via `cmdToTaskList`, and then compare those.

The issue is that `cmdToTaskList` now takes a `Dispatcher` as an argument. So, we now can compare two commands for equality only where there's a `Dispatcher` in scope.

Now that I've slept it over, I think that this may actually kill this PR. The whole point of this PR was to not need a `Dispatcher` in scope in unexpected places.